napari_allencell_annotator/util/file_utils.py

```diff
@@ -51,10 +62,66 @@ def get_sorted_files_in_dir(dir_path: Path) -> list[Path]:
         dir_path: list[Path]
             The path to a directory
         """
-        return sorted(list(dir_path.glob("*.*")))
+
+        return sorted(list(dir_path.glob("*")))
 
     @staticmethod
     def shuffle_file_list(files: list[Path]) -> list[Path]:
+        """
+        Shuffles the file list.
+
+        Parameters
+        ----------
+        files: list[Path]
+            A file list to be shuffled
+
+        Returns
+        -------
+        list[Path]
+            The shuffled file list
+        """
         shuffled_list = files.copy()
         random.shuffle(shuffled_list)
         return shuffled_list
+
+    @staticmethod
+    def get_file_name(path: Path):
```

Comment:
return type

Reply:
added in https://github.com/aics-int/napari-allencell-annotator/pull/79/commits/5d6c4d38d51fab0f3a1af0cbdb96244fa25b0cf6